### PR TITLE
Fix/pid max warning

### PR DESCRIPTION
**Pull request by artursartamonovsadi**
Set platform default max values for pid_max

### CODE REVIEW

**Comment by vasbimpikasadi** [comment]
I think that should only be included in the minimal image? Does it reproduce on the full image, too?

**Comment by artursartamonovsadi** [comment]
/proc/kernel/pid_max values is set inside Linux Kernel there shouldn't be any difference in boot method as kernel configs are same for any boot method max value is define by:
#define PID_MAX_LIMIT (CONFIG_BASE_SMALL ? PAGE_SIZE * 8 : \
	(sizeof(long) > 4 ? 4 * 1024 * 1024 : PID_MAX_DEFAULT))
so its depends on CONFIG_BASE_SMALL and PAGE_SIZE

**Review by vasbimpikasadi — APPROVED** [review]

File: meta-adi-adsp-sc5xx/recipes-core/systemd/systemd_250.4.bbappend [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FILESEXTRAPATHS:append := "${THISDIR}/${PN}:"
+
+SRC_URI:append = "file://90-override.conf"
+
+do_install:append() {
+	echo "Trying ot install override"
```
Comment (UtsavAgarwalADI) [inline meta-adi-adsp-sc5xx/recipes-core/systemd/systemd_250.4.bbappend]: Maybe this is a typo?

**Review by UtsavAgarwalADI — APPROVED** [review]
cat /proc/sys/kernel/pid_max gives correct value (32768)

**Pull request merged**